tests/ui/UnreadIndicatorsTest.js

```diff
@@ -0,0 +1,521 @@
+                5: TestHelper.buildTestReportComment(USER_B_EMAIL, 5, MOMENT_TEN_MINUTES_AGO.add(50, 'seconds').unix()),
+                6: TestHelper.buildTestReportComment(USER_B_EMAIL, 6, MOMENT_TEN_MINUTES_AGO.add(60, 'seconds').unix()),
+                7: TestHelper.buildTestReportComment(USER_B_EMAIL, 7, MOMENT_TEN_MINUTES_AGO.add(70, 'seconds').unix()),
+                8: TestHelper.buildTestReportComment(USER_B_EMAIL, 8, MOMENT_TEN_MINUTES_AGO.add(80, 'seconds').unix()),
+                9: TestHelper.buildTestReportComment(USER_B_EMAIL, 9, MOMENT_TEN_MINUTES_AGO.add(90, 'seconds').unix()),
+            });
+            Onyx.merge(ONYXKEYS.PERSONAL_DETAILS, {
+                [USER_B_EMAIL]: TestHelper.buildPersonalDetails(USER_B_EMAIL, USER_B_ACCOUNT_ID, 'B'),
+            });
+
+            // We manually setting the sidebar as loaded since the onLayout event does not fire in tests
+            AppActions.setSidebarLoaded(true);
+            return waitForPromisesToResolve();
+        })
+        .then(() => renderedApp);
+}
+
+describe('Unread Indicators', () => {
+    afterEach(Onyx.clear);
+
+    it('Display bold in the LHN for unread chat and new line indicator above the chat message when we navigate to it', () => {
+        let renderedApp;
+        return signInAndGetAppWithUnreadChat()
+            .then((testInstance) => {
+                renderedApp = testInstance;
+
+                // Verify no notifications are created for these older messages
+                expect(LocalNotification.showCommentNotification.mock.calls.length).toBe(0);
+
+                // Verify the sidebar links are rendered
+                const sidebarLinks = renderedApp.queryAllByA11yLabel('List of chats');
+                expect(sidebarLinks.length).toBe(1);
+                expect(isDrawerOpen(renderedApp)).toBe(true);
+
+                // Verify there is only one option in the sidebar
+                const optionRows = renderedApp.getAllByA11yHint('Navigates to a chat');
+                expect(optionRows.length).toBe(1);
+
+                // And that the text is bold
+                const displayNameText = renderedApp.getByA11yLabel('Chat user display names');
+                expect(lodashGet(displayNameText, ['props', 'style', 0, 'fontWeight'])).toBe(fontWeightBold);
+
+                return navigateToSidebarOption(renderedApp, 0);
+            })
+            .then(() => {
+                // Verify that the report screen is rendered and the drawer is closed
+                expect(isDrawerOpen(renderedApp)).toBe(false);
+
+                // That the report actions are visible along with the created action
+                const createdAction = renderedApp.getByA11yLabel('Chat welcome message');
+                expect(createdAction).toBeTruthy();
+                const reportComments = renderedApp.getAllByA11yLabel('Chat message');
```

Comment:
Was there a specific reason to keep these as `getBy` instead of switching to `queryBy`?

Reply:
Nope! I think we are good to update them all.